shared/src/protocol/navigation/location.rs

```diff
@@ -0,0 +1,82 @@
+use std::fmt::Display;
+
+use euclid::{approxeq::ApproxEq, Point2D};
+use num_traits::FromPrimitive;
+use serde::{Deserialize, Serialize};
+
+use crate::solar_systems::{self, Solar};
+
+#[derive(Debug, Copy, Hash, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub struct SolarSystemId(pub i64);
```

Comment:
This name implies its an ID fo the Solar System, not for a location within the solar system. The orignial name was SolarSystemLocationId -- based on the naming of these modules, I think this should just be `Id`, and usage should be changed to `location::Id`

Reply:
Done.